packages/react-router-config/modules/renderRoutes.js

```diff
@@ -4,10 +4,16 @@ import Route from 'react-router/Route'
 
 const renderRoutes = (routes, extraProps = {}) => routes ? (
   <Switch>
-    {routes.map((route, i) => (
-      <Route key={i} path={route.path} exact={route.exact} strict={route.strict} render={(props) => (
-        <route.component {...props} {...extraProps} route={route}/>
-      )}/>
+    {routes.map(route => (
+      <Route
+        key='prevent-remount-and-use-static-key'
```

Comment:
Can you make this shorter? Seems a waste to put a long string here just to remind devs. `key='fixed'` would also work, no?